pom.xml

```diff
@@ -209,10 +210,12 @@
         <dependencies>
             <dependency>
                 <groupId>io.netty</groupId>
-                <artifactId>netty-handler</artifactId>
-                <version>4.1.107.Final</version>
+                <artifactId>netty-bom</artifactId>
```

Comment:
why did the artifact id change? Maybe netty is not used at all?

Reply:
several different netty dependencies are being used. Only netty-handler's version is being overrided (although all of them should be the same version). I've replaced this with the netty-bom to synchronize all netty dependencies to that version.